pages/parent-enrollment.tsx

```diff
@@ -0,0 +1,121 @@
+import Wrapper from "@components/SDCWrapper";
+import {
+    Flex,
+    Box,
+    Button,
+    Stack,
+    Center,
+    Text,
+    useRadio,
+    useRadioGroup,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { CloseButton } from "@components/CloseButton";
+
+function RadioCard(props) {
+    const { getInputProps, getCheckboxProps } = useRadio(props);
+
+    const input = getInputProps();
+    const checkbox = getCheckboxProps();
+    // Returns one radio card with the child's name. Button changes style when selected
+    return (
+        <Box as="label">
+            <input {...input} />
+            <Box
+                {...checkbox}
+                lineHeight="24px"
+                fontSize="16px"
+                fontWeight="normal"
+                textColor="#0C53A0"
+                width="340px"
+                height="50px"
+                border="2px solid #E1E1E1"
+                borderRadius="6px"
+                _checked={{
+                    fontWeight: "normal",
+                    bg: "white",
+                    textColor: "#0C53A0",
+                    border: "2px solid #0C53A0",
+                }}
+                textAlign="center"
+                py="13px"
+                my="10px"
+            >
+                {props.children}
+            </Box>
+        </Box>
+    );
+}
+
+export default function ParentEnrollClass(): JSX.Element {
+    // Next button is disabled by default, activates when a child is selected
+    const [selectedChild, setSelectedChild] = useState<string>("");
```

Comment:
RE: this @feng-eric @brandonwong368 
> I did notice one bug where the blue highlight won't appear until the second click.
> 
> You can see in the screenshot that the Next button is enabled but there isn't a blue outline
> ![image](https://user-images.githubusercontent.com/46411197/128110151-600d99c1-aede-4502-96d5-35a8aae775d1.png)
> 
> After second click:
> ![image](https://user-images.githubusercontent.com/46411197/128110139-96104009-ae5a-4b17-bf94-111b261f4e6e.png)

Hmm. I don't really know the root cause, but I noticed that if you move this call to `useState` to within the `ChildrenButtons` component, the box highlighting works fine (however, getting the parent component's state to update as well will require a bit more work). I can try to look into this more later but at least it's a start??

Reply:
Update: couldn't figure it out so we switched to buttons lol